frontend/src/v5/ui/routes/dashboard/teamspaces/teamspaceLayout/teamspaceLayout.component.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ *  Copyright (C) 2021 3D Repo Ltd
```

Comment:
For Daniel,  the Copyright year is correct. This file already existed, but it was moved to a different folder, so GitHub thinks it's a new file.